include/sound/sof/ext_manifest4.h

```diff
@@ -90,6 +90,18 @@ struct sof_man4_segment_desc {
 	uint32_t file_offset;
 } __packed;
 
+/* bit definition for type in sof_man4_module */
```

Comment:
the topology patches have been posted upstream @RanderWang. So these can no longer be fixups

Reply:
sure, thanks